package.json

```diff
@@ -80,7 +80,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
-    "@playwright/test": "^1.31.2",
+    "@playwright/test": "1.31.2",
```

Comment:
Is it still necessary to lock the version to `1.31.2`?

Reply:
No it was cause my local Node is too recent so it installs the wrong one, will revert the locking